packages/kitchen/src/components/Select/index.tsx

```diff
@@ -0,0 +1,130 @@
+import React from "react";
+import styled from "styled-components";
+import { KitchenComponent, NormalSizes } from "../../types";
+
+export type SelectProps = KitchenComponent & {
+  /**
+   * The size of the select.
+   * @default "normal"
+   */
+  size?: NormalSizes;
+  /**
+   * The label of the select.
+   */
+  label?: string;
+  /**
+   * The placeholder of the select.
+   */
+  placeholder?: string;
+  /**
+   * The prefix of the select.
+   */
+  prefix?: React.ReactNode;
```

Comment:
This is not a react node, this is a JSX Element, but the best type here is `SVGElement`